webapp/src/pages/Landing/Landing.tsx

```diff
@@ -7,10 +16,17 @@ export default function Landing() {
     <section className="Landing">
       <Icon
         type={IconType.PLAY}
-        color={Color.LIGHT}
+        color={Color.ONLINE}
         size={IconSize.LARGE}
       ></Icon>
-      <h1 className="text-style-0-bold">Landing 🚀</h1>
+      <Text
+        type={TextType.HEADER}
+        size={TextSize.LARGE}
```

Comment:
Wouldn't it make sense for "TextType.HEADER" to imply "TextSize.LARGE"? I fail to see why wouldn't we want to pack type together with size.

Reply:
I agree with @Damrod, It would make sense to have a default TextSize for each TextType

Reply:
nice!! I'll keep the weights since they are clearly defined in Figma and don't have to be tied to the text type